### ISSUE TITLE
No matching distribution found for opencv-python (from scenedetect==0.5.1)

### ISSUE THREAD

**Issue by jeffhoek** [Breakthrough/PySceneDetect#111]
I was able to install pyscenedetect prior to the 0.5.1 release using `pip install scenedetect`.  However I'm now running into this error after the 0.5.1 release:
```
pip install scenedetect
...
No matching distribution found for opencv-python (from scenedetect==0.5.1)
```

**Comment by Breakthrough** [comment]
Hi @jeffhoek;

Thanks for the report.  Can you share what operating system and version of Python you're using?

Thank you!

**Comment by Breakthrough** [comment]
I may need to change the setup.py file so you can do:

    pip install scenedetect[opencv]

Thus allowing manual installation of the OpenCV package.  I thought opencv-python was available for almost all platforms, thus my query.  Thanks!

**Comment by jeffhoek** [comment]
Thanks for the quick reply--sorry for not adding details.  I was attempting to add scenedetect into a custom Dockerfile based on https://hub.docker.com/r/ondrejit/micro-opencv-ffmpeg.  This image has ffmeg 4.0.1, OpenCV 3.4.1, and Python 3.6.  I will try out your fix.

**Comment by Breakthrough** [comment]
Gotcha @jeffhoek - if that's the case then the new installer should fix that up for you, since originally PySceneDetect didn't have OpenCV listed as a dependency for issues like this.

Let me know how you make out with the new installer (already live on `pip`) so I can verify that this resolved the issue.

**Comment by Breakthrough** [comment]
Closed by [commit 42aaca2](https://github.com/Breakthrough/PySceneDetect/commit/42aaca2ea43aa7bcea44ed58f5f31e67f2d935fb).  I believe making it an optional dependency will fix the original issue you were running into.  If you are still experiencing issues, please feel free to respond here or create a new issue.

Thank you again for your submission and bringing this to my attention!

**Comment by jeffhoek** [comment]
I am no longer using the image.
